aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/ServletLambdaContainerHandlerBuilder.java

```diff
@@ -165,7 +165,7 @@ public Builder responseTypeClass(Class<ResponseType> responseType) {
     /**
      * Uses an async initializer with the given start time to calculate the 10 seconds timeout.
      *
-     * @deprecated As of release 1.5 this method is deprecated in favor of the parameters-less one {@link ServletLambdaContainerHandlerBuilder#asyncInit()}.
+     * @deprecated As of release 2.1 this method is deprecated. Initializer is always async if running in on-demand.
```

Comment:
should be 2.0.0 (currently we just have 2.0.0-M1 and 2.0.0-M2 milestone releases out there)